fs-extra/fs-extra.d.ts

```diff
@@ -28,6 +28,10 @@ declare module "fs-extra" {
 	export function mkdirsSync(dir: string, options?: MkdirOptions): void;
 	export function mkdirpSync(dir: string, options?: MkdirOptions): void;
 
+	export function move(src: string, dest: string, callback?: (err: Error) => void): void;
+	export function move(src: string, dest: string, filter: CopyFilter, callback?: (err: Error) => void): void;
+	export function move(src: string, dest: string, options: CopyOptions, callback?: (err: Error) => void): void;
```

Comment:
The docs you linked suggest that `move` and `copy` actually have different options.